deployment/updates/scripts/xml/notifications/explicit_live_notification.template.xml

```diff
@@ -25,7 +25,7 @@
 				<eventConditions objectType="array">
 					<item objectType="KalturaEventObjectChangedCondition">
 						<description>Entry view mode or recording status changes</description>
-						<modifiedColumns>{view_mode,recording_status}</modifiedColumns>
+						<modifiedColumns>view_mode,recording_status</modifiedColumns>
```

Comment:
Did you meant to merge this change along with the url modification? 
It would be better to separate to 2 different pull requests.

Reply:
you are correct, will do